Gemfile.lock

```diff
@@ -173,7 +173,7 @@ GEM
     nokogiri (1.10.1)
       mini_portile2 (~> 2.4.0)
     parallel (1.14.0)
-    parser (2.6.0.0)
+    parser (2.6.2.0)
```

Comment:
Probably good to not commit your Gemfile.lock if you didn't actually add any gems yourself